docs/terms/validation_result.md

```diff
@@ -1,15 +1,91 @@
+
+### Relationship to other objects
+
+Validation Results are generated inside a Checkpoint when a <TechnicalTag relative="../" tag="validator" text="Validator" /> runs an Expectation Suite against its paired <TechnicalTag relative="../" tag="batch_request" text="Batch Request" />.  The Validation Result will be passed to the Checkpoint's `action_list` where it will be available to any Actions that may need it.  The `StoreValidationResultAction` subclass of `ValidationAction` will save Validation Results to the <TechnicalTag relative="../" tag="validation_result_store" text="Validation Result Store" /> if it is present in a Checkpoint's `action_list`.
+
+## Use cases
+
+<CreateHeader/>
+
+A Validation Result is generated when you use [the interactive workflow to create Expectations](../guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md) and the newly created Expectation is run against the sample <TechnicalTag relative="../" tag="batch" text="Batch" /> of data you are working with.  These individual Validation Results will be created as an `ExpectationValidationResult` instance.
+
+<ValidateHeader/>
+
+Validation Results are generated when you run Checkpoints to Validate data.  Because Checkpoints operate on Expectation Suites rather than individual Expectations, the Validation Results generated by Checkpoints will be created as `ExpectationSuiteValidationResult` instances. The difference between an `ExpectationValidationResult` instance and an `ExpectationSuiteValidationResult` instance is that the `ExpectationSuiteValidationResult` will contain additional information pertaining to the Expectation Suite that was Validated, and will contain the data of an `ExpectationValidationResult` for each Expectation in the Expectation Suite.  These `ExpectationValidationResult` instances will be in a list accessed by the `results` attribute of the `ExpectationSuiteValidationResult` instance.
+
+The Validation Results generated as `ExpectationSuiteValidationResult` instances can be saved in JSON format in a Validation Result Store, and rendered as <TechnicalTag relative="../" tag="data_docs" text="Data Docs" />.  Both of these activities are handled by Actions in the Checkpoint's `action_list`.  In most cases, when you see "Validation Results" in Great Expectations, it will be in reference to the output generated as an `ExpectationSuiteValidationResult` when a Checkpoint is run.
```

Comment:
Again, I'm being pendantic. "Into" could also work.

```suggestion
The Validation Results generated as `ExpectationSuiteValidationResult` instances can be saved in JSON format in a Validation Result Store, and rendered in <TechnicalTag relative="../" tag="data_docs" text="Data Docs" />.  Both of these activities are handled by Actions in the Checkpoint's `action_list`.  In most cases, when you see "Validation Results" in Great Expectations, it will be in reference to the output generated as an `ExpectationSuiteValidationResult` when a Checkpoint is run.
```